app/controllers/concerns/decidim/collaborations/needs_collaboration.rb

```diff
@@ -55,8 +55,7 @@ def out_of_collaboration_period?
         end
 
         def detect_collaboration
-          request.env['current_collaboration'] ||
-            Collaboration.find_by(id: params[:collaboration_id] || params[:id])
+          Collaboration.find_by(id: params[:collaboration_id] || params[:id])
```

Comment:
Why this?

Reply:
Is a `current_collaboration` variable set in the request environment anywhere? I don't recall features ever doing this. I was investigating the spec failures and I removed it while at it, fearing that the collaboration from DB was somehow not being loaded because of this.

Reply:
yes, you are right, it is not set anywhere else :+1: